native-midi/app/src/main/cpp/AppMidiManager.cpp

```diff
@@ -148,13 +148,12 @@ extern "C" {
 void Java_com_example_nativemidi_AppMidiManager_startReadingMidi(
         JNIEnv* env, jobject, jobject midiDeviceObj, jint portNumber) {
 
-    media_status_t status;
```

Comment:
Shouldn't errors at least be logged?

Reply:
This is purely to silence a `-Werror` from a variable that is set but never used. There should be no change in behavior.

Reply:
I know, but the change just makes the symptom go away, not the problem. Is there some reason we shouldn't be handling these error cases? If so, that's worthy of a comment.

Reply:
Well, I didn't look too deeply, but there didn't seem to be an obvious way to do something better with the error.

Now, as I look more closely, this is already not the only instance of swallowing statuses. For example, I just spotted this gem on line 177:

`/*media_status_t status =*/ AMidiDevice_release(sNativeReceiveDevice);`

I mean, I can add a TODO if you want, but actually improving the functionality of the samples is not the point of this change, and I'm not ready to go down that rabbit hole just yet.

Reply:
Filing a bug would work. I just know that if we don't do anything, we will forget about this.

Reply:
https://github.com/android/ndk-samples/issues/870